movo_robot/movo_bringup/launch/movo1.launch

```diff
@@ -13,15 +13,18 @@
         name="sensors_bringup" output="screen"/>
         
     <!-- Platform Manipulation -->
-    <!-- Ben TODO Uncomment
     <node pkg="si_utils" type="timed_roslaunch"
         args="12 movo_bringup movo_manipulation.launch local:=false"
-        name="manipulation_bringup" output="screen"/> -->
+        name="manipulation_bringup" output="screen"/>
 
     <!-- Platform Teleoperation -->
     <node pkg="si_utils" type="timed_roslaunch"
         args="14 movo_bringup movo_teleop.launch local:=false"
         name="teleop_bringup" output="screen"/>
-           
+
+    <!-- Moveit bringup-->
+    <node pkg="si_utils" type="timed_roslaunch"
+        args="20 movo_bringup movo_moveit.launch local:=true"
+        name="movo_moveit_bringup" output="screen"/>           
```

Comment:
Do we want to keep moveit on movo1? has anyone checked if this has improved performance?

Reply:
lets leave it on movo2 for now. lets do the perf test after v0. we wanna get v0 done with asap and i think this is non-blocking